src/lib.rs

```diff
@@ -316,6 +316,26 @@ impl Check {
             .deps(false)
             .silence(!config.is_verbose());
 
+        // If both the current and baseline rustdoc are given explicitely as a file path, we don't need to use the installed rustc, and this check can be skipped.
```

Comment:
```suggestion
        // If both the current and baseline rustdoc are given explicitly as a file path, we don't need to use the installed rustc, and this check can be skipped.
```
Typo fix + let's try to keep comments and string literals to below 100 chars both here and below